src/phone-validation.js

```diff
@@ -0,0 +1,17 @@
+export default {
```

Comment:
Good idea to make this shareable!

Instead of putting this directly in `src/`, could you create `src/utils/` and move it in there?

Reply:
Done!

Also, I've added a line to convert the number to the format ###-###-#### before sending it to the server, since we haven't merged https://github.com/UPchieve/server/pull/99 yet.

Reply:
@dmnisson I'm getting an error when I try saving the profile of a student user -- looks like it does the phone conversion regardless of volunteer status.

I just merged https://github.com/UPchieve/server/pull/99 so you can safely remove that check

Reply:
This has been removed.